src/components/biggive-main-menu/biggive-main-menu.scss

```diff
@@ -0,0 +1,464 @@
+:host {
+	display: block;
+}
+
+/* Googlefont Poppins CDN Link */
+// @import "https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap";
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	font-family: 'Euclid Triangle', sans-serif;
+}
+
+.arrow {
+	margin-left: 5px;
+	line-height: 70px;
+	text-align: center;
+	display: inline-block;
+	color: black;
+	transition: all 0.3s ease;
+}
+
+a {
+	// target all classes prefixed with 'icon-' so we don't repeat code
+	&[class^='icon-'],
+	&[class*='icon-'] {
+		padding-left: 20px;
+		background-repeat: no-repeat;
+		background-size: auto 10px;
+		background-position: left center;
+	}
+
+	&.icon-green {
+		background-image: url(/assets/images/triangles-combined.svg);
+	}
+
+	&.icon-children {
+		background-image: url(/assets/images/triangles-combined-children.svg);
+	}
+
+	&.icon-christmas {
+		background-image: url(/assets/images/triangles-combined-christmas-challenge.svg);
+	}
+
+	&.icon-green-match {
+		background-image: url(/assets/images/triangles-combined-green-match.svg);
+	}
+
+	&.icon-women-girls {
+		background-image: url(/assets/images/triangles-combined-women-girls.svg);
+	}
+
+	&.icon-anchor-match {
+		background-image: url(/assets/images/triangles-combined-anchor.svg);
+	}
+
+	&.icon-emergency {
+		background-image: url(/assets/images/triangles-combined-emergency.svg);
+	}
+}
+
+.mobile-only {
+	display: none !important;
```

Comment:
Could we achieve the same using more specific selectors and avoid `!important`? This is usually more maintainable.

This goes for all the other `!important`s in this file too.

Reply:
@NoelLH I've removed what I can. There are now only three occurrences left, two of which are used in the `.mobile-only` css class. But I think these two are totally fine because I designed that css class to be for general-purpose, so it can be applied anywhere. For such a use case, it makes sense to use `!important` instead of tight scoping, as this ensures the css class can be used flexibly in multiple places, and if you check the story file you'll see this class is used in two differing places.

Other than that, there's one final occurrence of `!important` for the `.display-sub-menu` class. It's proven to be quite fiddly to get rid of, and quite time consuming to do. I think it's not worth the additional time, especially as a highly doubt there will be huge menu changes moving forward at least in the near future. I think let's prioritise rolling out a new working menu instead of investing time into this.

cc @bdsl